bridge/cypress/integration/sequences.spec.ts

```diff
@@ -152,6 +152,12 @@ describe('Sequences', () => {
     cy.get('.dt-filter-field-input ').type('{esc}');
   }
 
+  it('should show remediation in regular state while running', () => {});
```

Comment:
Please remove the empty tests, as they are the same as for `remediation-sequences.spec.ts`